.github/workflows/ci.yaml

```
@@ -1,57 +1,59 @@
-name: deploy-rvc
+name: deploy-rvc-terraform
```

```suggestion
name: deploy-rvc
```

I don't think you need to add a `terraform` suffix to the deployment workflow name.